wrappers/wrapper.rb

```diff
@@ -745,6 +811,46 @@ def simplify_constraints(vrp)
       vrp
     end
 
+    def empty_result(solver, vrp, unassigned_reason = nil)
+      {
+        solvers: [solver],
+        cost: nil,
+        costs: Models::Costs.new({}),
+        iterations: nil,
+        routes: vrp.vehicles.collect{ |vehicle| { vehicle_id: vehicle.id, activities: [] } },
+        unassigned: (vrp.services.collect{ |service|
+          {
+            service_id: service.id,
+            type: service.type.to_s,
+            point_id: service.activity.point_id,
+            detail: build_detail(service, service.activity, service.activity.point, nil, nil, nil),
+            reason: unassigned_reason
+          }.delete_if{ |_k, v| !v }
+        }) + (vrp.shipments.collect{ |shipment|
+          [{
+            shipment_id: shipment.id,
```

Comment:
The standard is to have pickup_shipment_id and delivery_shipment_id instead of shipment_id + type.

The current version in ortools.rb is about to be fixed

It should follow the definition given in api/v01/entities/vrp_result.rb

Reply:
https://github.com/braktar/optimizer-api/commit/fe562aa2bdf6dd8336d7037a34c3bc42707f39e7